userguide/tutorials/development.adoc

```diff
@@ -1,127 +1,501 @@
+               <goal>maven-version</goal>
+               <goal>add-source</goal>
+               <goal>parse-version</goal>
+            </goals>
+         </pluginExecutionFilter>
+         <action>
+            <execute>
+               <runOnConfiguration>true</runOnConfiguration>
+               <runOnIncremental>true</runOnIncremental>
+            </execute>
+         </action>
+      </pluginExecution>
+      <pluginExecution>
+         <pluginExecutionFilter>
+            <groupId>com.hubspot.maven.plugins</groupId>
+            <artifactId>dependency-scope-maven-plugin</artifactId>
+            <versionRange>0.10</versionRange>
+            <goals>
+               <goal>check</goal>
+            </goals>
+         </pluginExecutionFilter>
+         <action>
+           <ignore />
+         </action>
+      </pluginExecution>
+      <pluginExecution>
+         <pluginExecutionFilter>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <versionRange>3.1.2</versionRange>
+            <goals>
+               <goal>analyze-duplicate</goal>
+               <goal>analyze-only</goal>
+            </goals>
+         </pluginExecutionFilter>
+         <action>
+            <ignore />
+         </action>
+      </pluginExecution>
+   </pluginExecutions>
+</lifecycleMappingMetadata>
 
-Pre-requisites:
+. Go to `Window > Preferences > Maven > Lifecycle Mappings`.Click *Browse* and select the path of the `lifecycle-mapping-metadata.xml` file. Click `Apply and Close`
 
-* JDK 7 or later installed
-* Maven 3.5.2 or later installed
-* Git 2.15.1 or later installed
+. Build the project again (In Project Explorer, `Right click on killbill > Maven > Update Project > OK`)
 
-=== Eclipse
+. This will get rid of all the build errors. Your Eclipse workspace should look like this:image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/development/eclipse_workspace.png[align=center]
 
-The Kill Bill projects can be imported into the Eclipse IDE as long as the https://www.eclipse.org/m2e/[m2e] plugin is installed. Recent versions of the https://www.eclipse.org/downloads/eclipse-packages/[Eclipse IDE for Java EE Developers] come with this plugin already.
 
-Some of the Maven plugins used may not be available to m2e, however, and the projects will not import completely. To work around this issue, the Eclipse `Preferences > Maven > Lifecycle Mappings` configuration can be updated to include mappings like the following:
+=== Configuring the database
+
+Before you can execute the code, you need to configure the Kill Bill database manually. By default Kill Bill expects *MySQL*, but you can also use *PostgreSQL*
+
+==== MySQL configuration
+
+In order to configure MySQL, you need to follow the steps given below:
+
+. Create a database using the following command:
+
+    create database killbill;
+   
+. Run the Kill Bill DDL https://docs.killbill.io/latest/ddl.sql[here].
```

Comment:
As discussed on Slack, let's add more step-by-step instructions here as well for MySQL.

Reply:
Done